average.js

```diff
@@ -1,6 +1,9 @@
 
 function average(numbers) {
-  return numbers.reduce((p, c)=> p + c, 0) / numbers.length;
+  numbers = numbers.filter(function(val) {
```

Comment:
The variable `numbers` means different things in different places - it is passed as the entire array but used as a filtered set inside the function.
Idea of naming variables is to make their intent clear without having to read the entire code. Try it out.